integration/has_vm/has_vm_test.go

```diff
@@ -0,0 +1,132 @@
+package has_vm_test
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	testhelperscpi "github.com/maximilien/bosh-softlayer-cpi/test_helpers"
+	slclient "github.com/maximilien/softlayer-go/client"
+	datatypes "github.com/maximilien/softlayer-go/data_types"
+	softlayer "github.com/maximilien/softlayer-go/softlayer"
+	testhelpers "github.com/maximilien/softlayer-go/test_helpers"
+)
+
+const configPath = "test_fixtures/cpi_methods/config.json"
+
+var _ = Describe("BOSH Director Level Integration for has_vm", func() {
+	var (
+		err error
+
+		client softlayer.Client
+
+		username, apiKey string
+
+		accountService      softlayer.SoftLayer_Account_Service
+		virtualGuestService softlayer.SoftLayer_Virtual_Guest_Service
+
+		virtualGuest  datatypes.SoftLayer_Virtual_Guest
+		createdSshKey datatypes.SoftLayer_Security_Ssh_Key
+
+		rootTemplatePath, tmpConfigPath, strVGID string
+
+		replacementMap map[string]string
+
+		o map[string]interface{}
```

Comment:
`o` is a pretty terrible name for any variable... Let's use a more descriptive name please.